mlir/test/Dialect/Vector/vector-rewrite-narrow-types.mlir

```diff
@@ -234,6 +234,72 @@ func.func @aligned_extsi_2d(%a: vector<8x32xi4>) -> vector<8x32xi32> {
   return %0 : vector<8x32xi32>
 }
 
+// CHECK-LABEL: func.func @aligned_extsi_i2_to_i8(
+func.func @aligned_extsi_i2_to_i8(%a: vector<8xi2>) -> vector<8xi8> {
```

Comment:
[nit] I'd be tempted to move `@aligned_extsi_i2_to_i8` next to `@aligned_extsi_i4_to_i8`. Same for other tests.